src/Account/LogIn.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import {
+  TextInput,
+  View,
+  KeyboardAvoidingView,
+  Alert,
+  TouchableHighlight,
+  Text,
+} from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import { FontAwesome5 } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import loginUser from '../redux/account/actionCreator';
+import styles from './styles';
+import Row from '../common/Row';
+import Column from '../common/Column';
+
+const SignInButton = ({ onPress }) => (
+  <TouchableHighlight style={styles.button} onPress={onPress} underlayColor="#5d70ba" activeOpacity={0.5}>
+    <Text style={styles.buttonText}><FontAwesome5 name="arrow-right" size={16} /></Text>
+  </TouchableHighlight>
+);
+
+const SignIn = () => {
```

Comment:
I know signin and login are the same, but we should use them consequently. Now the filename and the component name is not matching. Pls forget the SignIn and use `Login` everywhere. Like LoginButton, Login.jsx etc...